src/codec/decoder/audio.rs

```diff
@@ -10,6 +10,11 @@ use crate::{
 pub struct Audio(pub Opened);
 
 impl Audio {
+	#[deprecated(
```

Comment:
I'm not super convinced about this and the addition `send_packet` and friends, I feel like `decode` should just adapt to the non-deprecated variants.

What's your thought on this?

Reply:
Yeah, makes sense.